### ISSUE TITLE
Source Code Pro font not working properly

### ISSUE THREAD

**Issue by jsgv** [Homebrew/homebrew-cask-fonts#3972]
After the switch from adobe fonts to google fonts for Source Code Pro it reintroduced a similar bug that had happened before. Text shows no color.
https://github.com/Homebrew/homebrew-cask-fonts/commit/598079999344b6e8783aad4eb3d676ff303701f9#diff-d3fce192d394c65db3038c0da2aafceb4550fa374d7c2ce0a86e43d220f8c7fe
https://github.com/Homebrew/homebrew-cask-fonts/commit/5d66fbd79267b8d0631e4cdb3a561f537c1a60a3#diff-d3fce192d394c65db3038c0da2aafceb4550fa374d7c2ce0a86e43d220f8c7fe

Reference:
https://github.com/Homebrew/homebrew-cask-fonts/issues/2639

Original issue on adobe repo that got patched:
https://github.com/adobe-fonts/source-code-pro/issues/250

Working
![working](https://user-images.githubusercontent.com/2798097/116217972-5d51ea00-a74a-11eb-8f93-beec9656af91.png)

Not working
![not-working](https://user-images.githubusercontent.com/2798097/116217976-5e831700-a74a-11eb-90ec-fffe6121debf.png)

**Comment by iandol** [comment]
Unfortunately, the google versions are old and do not contain many fixes and updates. The PR to restructure the names: https://github.com/Homebrew/homebrew-cask-fonts/pull/3898 led to an understandable decision to remove all adobe fonts (https://github.com/Homebrew/homebrew-cask-fonts/pull/3903) and they got replaced with the older Google ones. Personally this regression is worse than the annoyance of the changed names, as while it is possible to substitute font names easily, having to deal with finding completely new fonts for missing glyphs or working round the bugs fixed over the past few years is a harder proposition. 

It is hard work for the volunteers that keep homebrew running, and I appreciate their hard work dealing with unsympathetic Corporations like Adobe, so I respect their choices and I think if the brew versions of these fonts don't work for you, we endusers should probably just manually manage Adobe fonts (I've removed the brew versions and manually updated already). But if there was a better solution that would be great!

**Comment by carlocab** [comment]
You can try hosting the removed fonts in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

This tap might also be a good candidate for inclusion in the list at https://docs.brew.sh/Interesting-Taps-and-Forks.

**Comment by vitorgalvao** [comment]
> You can try hosting the removed fonts in your own tap.

That was going to be my suggestion. That’s the way to go in this case, or convince Google Fonts to merge the current PRs.

**Comment by hcguersoy** [comment]
Same issue here.

But closing the issue with a "try hosting yourself" is IMHO not the expected answer nor a solution.

IMHO this issue should be reopened until it is fixed. 
Or remove it completely but not offer something broken.

**Comment by carlocab** [comment]
I'm sorry, but I didn't realise we were obligated to give expected answers. Could you tell me how we came to incur this obligation?

**Comment by hcguersoy** [comment]
I didn't tell that you or someone else was "obligated" to do something. I just told you my expectations.
And I have not yet realised that the homebrew community has changed to a two class society: "members" and "users".
Cannot understand your answer otherwise.

I would suggest to remove something broken (in this case this font) or fix it. 
If you are not able to fix it, which should be case here due to the broken upstream stuff, remove it.

**Comment by carlocab** [comment]
Yes, but why are your expectations even relevant here? It seems as if your expectations should form some sort of impetus for us to act. Cannot understand your answer otherwise.

I would suggest that you consider how much work you're asking other people to do for you. If you want something done, please either:
a. open a pull request so it can be reviewed; or,
b. contribute to hosting the fonts you need in a tap.

**Comment by vitorgalvao** [comment]
> And I have not yet realised that the homebrew community has changed to a two class society: "members" and "users".

I have no idea what you’re going on about. Are you talking about the badges on the comments? That’s a GitHub automatic feature, we have no control over it. It indicates that @carlocab is a maintainer in the Homebrew organisation so users know, it has nothing to do with a “class society”.

I’m locking this because you’re coming off as combative and I’d rather not escalate further. Especially since you’ve never contributed with either code or discussion before, you shouldn’t be making demands on how to run the project.

It’s always the user with a short contributions graph which come guns blazing. That’s the kind of attitude that burns out open-source developers. We are *volunteers* running *for free* a project *you benefit from* in *our spare time*. I’m not even asking for gratitude, but a little consideration would be nice.

**Comment by vitorgalvao** [comment]
Note: in the past I used to never lock issues in these situations, because I thought it was unfair to not let the other person reply and possibly apologise publicly.

But after years of that strategy bearing no fruit, I realised it is unfair to myself, other maintainers, and the respectful users to *not* do what I can to preserve our mental sanity, which we need to run a project of this magnitude.
